Lib/Rabbit4000/STDVDRIVER.LIB

```diff
@@ -162,6 +162,7 @@ ioi	ld		(WDTCR), a				; now restart the secondary watch dog timer
 ; completion of the ISR to not be delayed past the next periodic interrupt by
 ; other interrupts.
 ;
+   align odd

```

Comment:
I think it would be beneficial to include comments here regarding the reasoning behind each `align` statement.  If you're aligning odd here to be even at some point, explain that.  Further down, explain which opcodes benefit from the even alignment.  This will make the code easier to maintain in the future -- if someone wants to alter these routines, they'll need to know exactly where the alignment is critical.

Reply:
`Done, though for the periodic ISR its mostly a more general improvement as several sequences of two byte instruction are helped - you could tune more but this was some low hanging fruit.